packages/mantine/src/components/sticky-footer/StickyFooter.tsx

```diff
@@ -27,7 +27,7 @@ export const StickyFooter: FunctionComponent<PropsWithChildren<StickyFooterProps
     return (
         <Paper className={classes.footer}>
             {borderTop ? <Divider size="xs" /> : null}
-            <Group position="right" spacing="xs" py="md" px="xl" {...others}>
+            <Group position="right" spacing="xs" pt="md" pb="md" px="xl" {...others}>
```

Comment:
Revert to `py`?

Reply:
For this one, I would suggest to use `pt="md" pb="md"` instead of `py=small`, because even though both of these do not impact the look of StickyFooter, but if we use `py=small`, then we will not be able to overwrite pt or pb individually by passing in props. In case of ModalWizard, since it already has padding, we need to disable the bottom padding of the StickyFooter

This shows ModalWizard with the proposed changes above:

<img width="867" alt="Screen Shot 2022-12-07 at 4 30 35 PM" src="https://user-images.githubusercontent.com/113382897/206300651-8b6c06c4-1275-42df-ad15-56db6b79d87e.png">

This shows what it looks like with `py=small`
<img width="846" alt="Screen Shot 2022-12-07 at 4 32 31 PM" src="https://user-images.githubusercontent.com/113382897/206300989-c0393577-ed0c-4caa-bfe7-30b863a66352.png">